osu.Game.Rulesets.Catch/UI/CatchPlayfield.cs

```diff
@@ -46,9 +46,12 @@ public override bool ReceivePositionalInputAt(Vector2 screenSpacePos) =>
 
         private readonly IBeatmapDifficultyInfo difficulty;
 
-        public CatchPlayfield(IBeatmapDifficultyInfo difficulty)
+        private readonly bool transferToCatcher;
```

Comment:
can this be an init property instead of a ctor boolean parameter? I really don't like the latter, they're horrible for readability

also needs better documentation